packages/jest-circus/src/run.ts

```diff
@@ -73,6 +74,9 @@ const _runTestsForDescribeBlock = async (
     let numRetriesAvailable = retryTimes;
 
     while (numRetriesAvailable > 0 && test.errors.length > 0) {
+      if (logErrorsBeforeRetry) {
+        console.error('Errors that caused Jest to retry test: ', test.errors);
```

Comment:
not a fan of using `console` here. Can we update the result object and reporter to include the information instead?